Formula/ppsspp.rb

```diff
@@ -24,8 +23,10 @@ class Ppsspp < Formula
   depends_on "python@3.10" => :build
   depends_on "libpng"
```

Comment:
It seems this is used for the headers only, so we can probably make it a build dependency. https://github.com/hrydgard/ppsspp/blob/52fd81129971def1b30df1533ebe8d42e16346cf/CMakeLists.txt#L985-L989

But maybe in a follow-up PR.